packages/framework/framework/src/useFluentTheme.ts

```diff
@@ -0,0 +1,6 @@
+import { defaultFluentTheme } from '@fluentui-react-native/default-theme';
+import { Theme, useTheme } from '@fluentui-react-native/theme-types';
+
+export function useFluentTheme(): Theme {
+  return useTheme() || defaultFluentTheme;
+}
```

Comment:
If we're introducing this, could we have this return the apple or android theme on each appropriate platform?

Reply:
Are the apple themes JS only? I think that's a reasonable discussion to have though this line keeps the functionality consistent with what we have been using, so I would split that out from this change.

Reply:
They are. The macOS one uses a `react-native-macos` specific API, but that's about it I think.